src/fragments/lib/push-notifications/react-native/migrate-from-previous-version.mdx

```diff
@@ -22,12 +22,21 @@ amplify pull
 npm uninstall @aws-amplify/pushnotification
 ```
 
+<Callout>
+
+Please note that you may need to delete your `yarn.lock` file, `package-lock.json` file as well as the `/node_modules` folder before adding the new native module.
```

Comment:
native modules? Or is the singular correct?

Should we say you may need? or do we just flat out recommend it?

Reply:
- Singular should be fine since its uninstalling one module and replacing with another.

- And actually flat recommendation is probably better